src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/Document.java

```diff
@@ -3,11 +3,11 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import javax.swing.*;
```

Comment:
is this a IDE thing? do we need *?

Reply:
It is actually unused. I thought I caught them all

Reply:
swish